addon/decorators/deactivated-account-handler.js

```diff
@@ -0,0 +1,14 @@
+export default function deactivatedAccountHandler(Class) {
+  return class DeactivatedAccountHandler extends Class {
+    setupController(controller, error) {
+      super.setupController(controller, error);
+
+      //TODO: Change that with variable based on environnement
+      const identityURL = 'https://user.upfluence.co/';
```

Comment:
The TODO is due in this PR or later?

Reply:
TODO is due in this PR.

When I remplace [this mixin](https://github.com/upfluence/ember-upf-utils/blob/master/addon/mixins/account-deactivated-error.js#L7) by this decoration, `getOwner(this).resolveRegistration('config:environment')` doesn't return right value.
Waiting to find a solution, so as not to be blocked, I used that

Reply:
Maybe extra this const outside ?

Reply:
I look at this